libs/metagen/src/mdk_rust/static/lib.rs

```diff
@@ -0,0 +1,2 @@
+// Copyright Metatype OÜ, licensed under the Mozilla Public License Version 2.0.
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
is this file required?

Reply:
Huh, that one's not supposed to be empty. Thanks.